dwhdelft.nl/pages/mixup.vue

```diff
@@ -0,0 +1,212 @@
+  </LayoutSmallHeader>
+
+  <section>
+    <PagesMixupLinkedEvents />
+  </section>
+
+  <section>
+    <div class="bg-brand-900 text-white text-lg mx-auto pt-12 pb-24 md:flex">
+      <div class="flex-1 px-4 lg:pr-32">
+        <div class="space-y-4">
+          <ElementsParagraphedText :paragraphs="t('intro')" class="md:text-xl md:leading-relaxed space-y-4" />
+          <p
+            v-if="barOpeningHours.announcement"
+            class="mt-3 mb-4 text-brand-900 bg-brand-450"
+            v-text="tt(barOpeningHours.announcement)"
+          />
+          <div
+            class="flex-1 rounded-lg shadow-xl bg-brand-450 md:text-xl p-4 relative w-full z-20 md:w-auto md:inline-flex items-center"
+          >
+            <div class="pt-6 md:pt-0 md:pl-4 space-y-2">
+              <p>{{ t('invite.announcement') }}</p>
+              <p class="font-bold">{{ t('invite.time', [barOpeningHours.start_time]) }}</p>
+            </div>
+          </div>
+          <div class="flex flex-1 space-x-4">
+            <div>
+              <a href="https://my.dwhdelft.nl/signup">
+                <ElementsSecondaryButton class="!text-brand-450" arrow>
+                  {{ t('membership_button') }}
+                </ElementsSecondaryButton>
+              </a>
+            </div>
+            <div>
+              <nuxt-link :to="localePath('barbuddy')">
+                <ElementsSecondaryButton class="!text-brand-450" arrow>
+                  {{ t('barbuddy_button') }}
+                </ElementsSecondaryButton>
+              </nuxt-link>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="hidden lg:block">
+        <div>
+          <img src="~/assets/images/photos/mixup/barnight.png" class="w-96" />
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="mixup-colors">
+    <div class="bg-[url('~/assets/images/photos/mixup/barvisual.jpg')] bg-cover bg-center py-10">
+      <ElementsContainer>
+        <div class="grid md:grid-cols-2 lg:grid-cols-4 gap-3">
+          <ElementsActionCard v-for="point in t('bulletPoints')" :key="point.title" :title="point.title">
+            <template #header>
+              <div class="w-full h-40 overflow-hidden mx-auto">
+                <img :src="imageOverviews(point.image)" class="object-cover w-full h-full" />
+                <img src="~assets/images/photos/mixup/barvisual.jpg" class="object-cover w-full h-full" />
+              </div>
+            </template>
+            <p class="-mt-3 text-brand-900 text-lg leading-snug" v-text="point.description" />
+          </ElementsActionCard>
+        </div>
+      </ElementsContainer>
+    </div>
+  </section>
+
+  <section>
+    <div class="bg-brand-900 pb-12">
+      <div class="mx-auto pt-12 pb-8">
+        <h1 class="text-center text-white font-medium text-5xl mb-6 leading-tight" v-html="t('highlights.title')" />
+      </div>
+      <div class="grid grid-cols-[1fr_minmax(0,1280px)_1fr]">
```

Comment:
II feel a bit mixed about the style of the highlights. It's a reuse of the ui components that we use to introduce people elsewhere, that is a kind of confusing ui pattern.

Maybe use an `ElementsActionCard` like on the Out-site's highlights? That way the images also come beter tot hun recht

Reply:
I see what you mean. (This was also a request from Colin, so I'll also discuss)
I do think the Outsite highlights look quite a bit like the cards above (with the 4 types of events), so this rounded element gives a nice difference in design.